### PR TITLE
added readme section regarding environment variables

### PR DESCRIPTION
**Pull request by rasmusebbesen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,20 @@ To make it easier to test Vapor apps locally on Linux using Docker, we have made
 - `Testing/Dockerfile-Swift4.1` which is using Swift 4.1 and is supposed to be used for Vapor 2 or Vapor 3 projects.
 - `Testing/Dockerfile-Swift4.2` which is using Swift 4.1 and is supposed to be used for Vapor 2 or Vapor 3 projects.
 
+## 🌳 Environment variables
+
+Environment variables are supported in Docker. Please read the [docs regarding environment variables](https://docs.docker.com/compose/environment-variables/) for info regarding how to configure them
```
Comment (siemensikkema) [inline README.md]:
```suggestion
Environment variables are supported in Docker. Please read the [docs regarding environment variables](https://docs.docker.com/compose/environment-variables/) for info regarding how to configure them.
```

File: README.md [hunk 2]
```diff
@@ -33,6 +33,20 @@ To make it easier to test Vapor apps locally on Linux using Docker, we have made
...
+
+In short composer prioritizes environment variables in the following order:
+
+ 1. Compose file
+ 2. Shell environment variables
+ 3. Environment file
+ 4. Dockerfile
+ 5. Variable is not defined
+
+If you want to use a `.env`-file, make sure to remove the environment section in your docker-compose file as described in the docs, referenced above.
```
Comment (siemensikkema) [inline README.md]:
```suggestion
If you want to use an `.env`-file, make sure to remove the environment section in your docker-compose file as described in the docs, referenced above.
```

File: README.md (hunk 1 above)
Comment (rasmusebbesen) [reply README.md]: Added the dot ;)

Comment (siemensikkema) [reply README.md]: Next time you can also just use the "commit suggestion" button 🙂

**Review by siemensikkema — APPROVED** [review]

**Pull request merged**